src/generators/android/templates/abstractSerializableClass.java.hbs

```diff
@@ -0,0 +1,8 @@
+package com.segment.generated;
```

Comment:
Can you add the autogenerated file warning here? We need it every file (we clear all generated files -- based on that header -- before re-building a client):

```java
/**
 * {{autogeneratedFileWarning}}
 */
```